odk1-src/central-install-digital-ocean.rst

```diff
@@ -190,14 +190,38 @@ ODK Central ships with a basic EXIM server bundled to forward mail out to the in
 
 .. _central-install-digital-ocean-sentry:
 
-Disabling Sentry
-----------------
+Disabling or customizing Sentry
```

Comment:
nit: Customizing

Reply:
As in delete the "Disabling or" and just leave "Customizing Sentry"?

Reply:
Taking a stab at this by changing to `Customizing Sentry`.